### PR TITLE
Added unit tests for SA1013 with refactoring

### LINKED ISSUE #199: Diagnostic unit tests: SA1013 (ClosingCurlyBracketsMustBeSpacedCorrectly)
Implement unit tests for #19.

### PR DESCRIPTION
**Pull request by Noryoko**
Included diagnostic for preceding spaces in interpolated strings.

fixes #199

### CODE REVIEW

File: StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1013ClosingCurlyBracketsMustBeSpacedCorrectly.cs [hunk 1]
```diff
@@ -60,12 +60,12 @@ private void HandleSyntaxTree(SyntaxTreeAnalysisContext context)
...
-                    break;
+                    case SyntaxKind.CloseBraceToken:
+                        this.HandleCloseBraceToken(context, token);
+                        break;
 
-                default:
-                    break;
+                    default:
+                        break;
                 }
```
Comment (Maxwe11) [inline StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1013ClosingCurlyBracketsMustBeSpacedCorrectly.cs]: :bulb:  replace `switch` with
    if (token.IsKind(SyntaxKind.CloseBraceToken))
    {
        HandleCloseBraceToken(context, token); 
    }

and make `HandleCloseBraceToken` static

**Comment by sharwell** [comment]
@Noryoko Try not to rebase, squash, or amend commits during the code review process. It makes it very difficult to see what changed between when the review process started and where we are now.

**Comment by Noryoko** [comment]
I'll keep that in mind for the future. Trying to get used to all the nuances. Thanks.

**Comment by sharwell** [comment]
:+1:

**Pull request merged**